app/konnect/analytics/custom-reports/use-cases.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Use Cases
+content_type: tutorial
+badge: plus
+---
+
+Custom reporting in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to CSV. 
+
+This tutorial guides you through the process of creating custom reports for key API metrics using {{site.konnect_saas}} custom reports. 
```

Comment:
```suggestion
This tutorial guides you through determining which custom report to create based on KPIs and the process of creating those custom reports for key API metrics. 
```
Not sure if I love the wording I suggested, I just wanted to get the point across that this tutorial both helps you figure out which report to create based on your KPIs and then we also guide you through how to create said reports.